### ISSUE TITLE
not working in electron windows.

### ISSUE THREAD

**Issue by dhananjay0610** [ealush/emoji-picker-react#378]
we are using <EmojiPicker> inside a project using react.js, we are using this inside a electron window and it isn't working.

**Comment by ealush** [comment]
What do you mean by not working?

**Comment by dhananjay0610** [comment]
the css isn't working

**Comment by ealush** [comment]
oh my. can you try importing the CSS separately from the dist just to see if it works?

**Comment by dhananjay0610** [comment]
inside dist, I just see index.js and index.js.map

**Comment by ealush** [comment]
Can you try: `dist/emoji-picker-react.cjs.development.css` if that works for you, I will try to understand how to provide a more reliable style source. I have little experience with electron, so I am not completely certain why it doesn't load the stylesheet.

**Comment by dhananjay0610** [comment]
i tried using
 import 'emoji-picker-react/dist/emoji-picker-react.cjs.development.css';

inside the file where <EmojiPicker is used but don't see any change.still same issue

**Comment by ealush** [comment]
Yeah, that might be an issue. The picker is build directly with CSS. What about trying to consume the stylisheet via a style tag as suggested here?

https://stackoverflow.com/questions/61162683/electron-how-to-import-or-include-css-installed-using-npm-package-manager-in-ou

**Comment by ealush** [comment]
I think this is no longer an issue since migrating to flairup
